examples/feature_selection/plot_feature_selection.py

```diff
@@ -61,17 +62,17 @@
 clf = svm.SVC(kernel='linear')
 clf.fit(X, y)
 
-svm_weights = (clf.coef_ ** 2).sum(axis=0)
-svm_weights /= svm_weights.max()
+svm_weights = (np.abs(clf.coef_)).sum(axis=0)
```

Comment:
Does normalizing this make more sense (dividing to make it unit norm?)

Reply:
I guess the code I corrected try to normalize the weights (by dividing all of them by the max). I think it's not a good way to process because we cannot compare the weights of the two models after that. For instance, if all my coefficients have the same importance, I'll get a normalize weight of 1 for each of them but the information we want is the contribution of each feature in the result who is 25%. So I guess dividing by the sum of all the weights is a better way to process to compare with another model.

Reply:
Since the data are not normalize, we should normalize the weights with the standard deviation of the data to be able to compare them. It will have a very little impact here (the features already have a similar std probably). I think that is make sense to take the absolute value but I am unsure about the `max` vs. `abs` normalization. IMO, we could divide by a constant (either `svm_weights.max()` or `svm_weights_selected.max()`) to have a max around 1 to make it easier to compare the univariate weights and classifier weights.

Reply:
remove the brackets around `np.abs`

Reply:
The purpose of this example is to point out univariate feature selection can increase the relevance of the informative features. So, what we want to show in the plot is the relevance of each feature for each model (with and without feature selection). In my opinion, the best way to measure the relevance of a feature is to compute the ratio between its weight and the sum of the weights of all the features of the model. The result we get is the percentage of importance of this feature in the corresponding model. Because the result is a percentage, we are able to compare the relevance of the feature between the different models.
That's why I guess divide by the sum is more appropriate to compare the relevance of each feature.

It's OK for the brackets, I'll correct that right now.